### PR TITLE
Resolve anticipated merge conflict between develop and main

### PR DESCRIPTION
**Pull request by tpdownes**
This PR resolves the merge conflict between main and develop that will arise after v1.15.0 is released.

### Submission Checklist

* [x] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [x] Are all tests passing? (`make tests`)
* [ ] Have you written unit tests to cover this change?
* [x] Is unit test coverage still above 80%?
* [x] Have you updated all applicable documentation?
* [x] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

**Comment by nick-stroud** [comment]
Since this is just an isolated change to develop, can we wait to execute it until we see that there is a merge conflict. Basically, leave this as draft until after release.

**Comment by tpdownes** [comment]
> Since this is just an isolated change to develop, can we wait to execute it until we see that there is a merge conflict. Basically, leave this as draft until after release.

Yes, that is the plan. Merge #1065 and then observe the merge conflict I anticipate will be resolved by #1066 (this PR).

**Review by nick-stroud — APPROVED** [review]
I think there are several ways to resolve the conflict. If this works then I see no harm so I am approving so you have that as a tool.

**Pull request merged**
